src/commands/EventCommand.js

```diff
@@ -419,15 +425,15 @@ class EventCommand extends SubsCommand {
             const events = startsEvents.concat(endsEvents).concat(goingEvents);
 
             let string = "";
-            if(events != null)
+            if (events != null)
                 events.forEach(event => {
                     string += "**" + event.name + "** - " + channel.guild.roles.find(r => r.id == this.eventModule.getMentionableRolesIds()[event.role]) + "\n";
```

Comment:
If we already have title, we can change all references to "name". Name should be just used for editing purposes.

Reply:
Please search for others references to name! I think we use it even in event annoucement.

Reply:
Can we make this one as a separate issue?